.codeclimate.yml

```diff
@@ -12,7 +12,7 @@ checks:
       threshold: 1000
   method-complexity:
     config:
-      threshold: 20
+      threshold: 30
```

Comment:
Ran into an issue where I hit a method complexity, let me know if this number provides value at all